RubberduckTests/QuickFixes/RemoveEmptyConditionBlockQuickFixTests.cs

```diff
@@ -526,13 +530,16 @@ End If
             var vbe = MockVbeBuilder.BuildFromSingleStandardModule(inputCode, out component);
             var state = MockParser.CreateAndParse(vbe.Object);
 
-            var inspection = new EmptyIfBlockInspection(state);
+            var inspection = new EmptyConditionBlockInspection(state, ConditionBlockToInspect.Else);
             var inspector = InspectionsHelper.GetInspector(inspection);
             var inspectionResults = inspector.FindIssuesAsync(state, CancellationToken.None).Result;
 
-            new RemoveEmptyIfBlockQuickFix(state).Fix(inspectionResults.First());
+            //Skip over empty Else block and get If block for the fix
+            var inspectionToFix = inspectionResults.ElementAt(1);
```

Comment:
Tests seem to be failing here (`.ElementAt(1)`) with an `IndexOutOfRangeException`. This means the **inspection** isn't yielding the expected results - these failing *quickfix* tests should be *inconclusive*, and there should be *at least* one failing inspection test.

Reply:
The refactor that uses `ConditionBlockToInspect.Else` is having only the results come back from the Else block. I'll get a failing test in to prove that it fail's appropriately and protect against false positives.